### PR TITLE
Updated the tox and travis configs -- BOM-1555

### PR DESCRIPTION
**Pull request by mraarif**
Relevant JIRA issue [here](https://openedx.atlassian.net/browse/BOM-1555).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,27 @@
...
-
+- codecov
 deploy:
   provider: pypi
   user: edx
   password:
     secure: xtak5/wlkNIWtIrY1kX9ht1xnL2Q7wMVHPofY2kEazILN8DJbwSqfNvwDV2E7mdevJK17o6dI1HDz7ZzzDAQaKudkN3tFIc/jZB9Cmxi/497NzOrhzshISCpnr+66r2+lR3bc2qsYH0W9w1LbVHzfWYYS4KhxBfCvSTA6mpxcwkgc38auoecnYa/SbqTgHhofmxg03E+Qb3czjoiloilS6sKNSuQdAbYB3nWVv/DsvDADkPSkqCXJh3J7dSrQS3oD9RHOg8Y5Rgzhqq/E2jIeTTU2lvaSplmkVBIy2jqi4Iys99a8hUPS6M9+ihsnn51znz58JJPUYZiMlQ5kiPzspMze0xhAD5ALpqCpvcJssFPumw4+M4MOcDZPdfzDlfEJne+IVP2mS7FJX2Tm1UWjlz/6vS+3DMJ6Xc0357GvjHcAmU8DaXep61n5jTapp5z/hZJcyYd+CsCDsvX3/Tl4u2dUTTFtHoYjRSq/gJ7kxgJYTDPEFMylN12IywSE5m2kPllFQCYT++2pgPlHuDbekpKhhTZ1iCmxjhvQ0FuqkGs1qkSftVtgY30sEpIMLSgiWJ7x3KMnmJUAh6bNyBmCUgFuEqU8+BRCF5C/hUrR5nKBmcroQECst2l6IgoXCMZOt8EAm2lIqKojANrvJmMVsNCd4thvJGdZ+TayG8b1Rc=
   distributions: sdist bdist_wheel
-  on:
+  true:
```
Comment (morenol) [inline .travis.yml:24]: Can you recheck that this is fine? @mraarif

Comment (mraarif) [reply .travis.yml]: updated.

File: tox.ini [hunk 2]
```diff
@@ -1,24 +1,20 @@
 [tox]
-envlist = py{27,35}-django{111}, py{35,36}-django{20,21,22},quality,docs
+envlist = , py{35,38}-django{22,30},quality,docs
```
Comment (morenol) [inline tox.ini:2]: Django 3.0 with Python3.5 is an invalid combination, also I think that there is an additional ,

File: tox.ini [hunk 3]
```diff
@@ -1,24 +1,20 @@
...
     -rtest_requirements.txt
-
-commands =
+commands = 
     python -Wd manage.py test django_sites_extensions --settings=test_settings --with-coverage --cover-package=django_sites_extensions
     coverage report
 
 [testenv:quality]
-commands =
+commands =
```
Comment (jmbowman) [inline tox.ini:17]: Can we fix the update script for `tox.ini` to not add these trailing spaces?  It doesn't match what we had before, and adds a fair bit of diff noise.

Comment (mraarif) [reply tox.ini:17]: sure, I was planning to update it not to add Django 3.0 for python 3.5

**Pull request merged**